templates/network.tftpl

```diff
@@ -1,25 +1,29 @@
-resource "azurerm_resource_group" "network" {
-  name     = "rg-$${local.subsciption_logical_name}Network-${stage}-01"
-  location = local.default_location
-  tags     = local.tags
+locals {
+  subnets = {
+  %{ for usecase, address_prefix in vnet_config.subnets ~}
+  ${usecase} = "${address_prefix}"
+  %{ endfor ~}
+}
```

Comment:
```suggestion
    %{ for usecase, address_prefix in vnet_config.subnets ~}
    ${usecase} = "${address_prefix}"
    %{ endfor ~}
  }
```
I think the white spacing is a bit off and we don't run a formatter after template generation